.golangci-kal.yml

```diff
@@ -32,8 +31,8 @@ linters:
               # - "optionalorrequired" # Every field should be marked as `+optional` or `+required`.
               # - "requiredfields" # Required fields should not be pointers, and should not have `omitempty`.
               # - "ssatags" # Ensure array fields have the appropriate listType markers
-              # - "statusoptional" # Ensure all first children within status should be optional.
-              # - "statussubresource" # All root objects that have a `status` field should have a status subresource.
+              - "statusoptional" # Ensure all first children within status should be optional.
+              - "statussubresource" # All root objects that have a `status` field should have a status subresource.
```

Comment:
Is this really effective?
As I mentioned in the issue, we have objects w/o status sub-resources.